app/src/main/java/com/example/habitsmasher/ui/profile/ProfileFragment.java

```diff
@@ -54,7 +55,13 @@ public View onCreateView(@NonNull LayoutInflater inflater,
                              ViewGroup container, Bundle savedInstanceState) {
         View view = inflater.inflate(R.layout.fragment_profile, container, false);
         // create the sample user
-        User user = new User("1", "TestUser", "123@gmail.com", "123", EMPTY_FOLLOWER_LIST, EMPTY_FOLLOWING_LIST);
+        User user = new User("1",
+                             "TestUser",
+                             "123@gmail.com",
+                             "123",
+                             EMPTY_FOLLOWER_LIST,
+                             EMPTY_FOLLOWING_LIST,
+                             EMPTY_REQUEST_LIST);
```

Comment:
I think this is old code. We no longer need this sample user in the profile fragment. Let's remove this.

- [ ] remove sample user